Formula/i386-elf-gcc.rb

```diff
@@ -0,0 +1,35 @@
+class I386ElfGcc < Formula
+  desc "The GNU compiler collection for i386-elf"
+  homepage "https://gcc.gnu.org"
+  url "https://mirrors.nju.edu.cn/gnu/gcc/gcc-8.2.0/gcc-8.2.0.tar.gz"
+  sha256 "1b0f36be1045ff58cbb9c83743835367b860810f17f0195a4e093458b372020f"
+  depends_on "gettext" => :build
```

Comment:
Blank line.
`gettext` and `xz` should not be necessary.
`gmp`, `mpfr`, `libmpc` and `i386-elf-binutils` are runtime deps, remove those `=> :build`